src/models/ItemFilterData.ts

```diff
@@ -1,30 +1,40 @@
+import { capitalize } from "lodash"
```

Comment:
Eventually, try to remove `lodash` and create the functions locally. Lodash is a big package and I think unnecessary nowadays.